### DISCUSSION TITLE
Possible to include Markdown files?

### DISCUSSION THREAD

**Discussion by nikitaourazbaev** [vuepress/core#262] [Q&A] [answered]
Will VuePress 2 support including Markdown files inside other Markdown files? The use cases I’m thinking of are: 

1. breaking down long files into fragments for easier editing and
2. displaying reusable snippets on several pages

Are there better solutions to these use cases?

**Comment by Mister-Hope** [comment]
It's a good feature request, it can be supported as a community plugin.

I may achieve it in my plugin `vuepress-plugin-md-enhance`, please track https://github.com/vuepress-theme-hope/vuepress-theme-hope/issues/1520.

**Comment by Mister-Hope** [comment, marked as answer]
```
@[md](path-to-md.md)

@[md{5-10}(path-to-md.md)
```

is suported in `vuepress-plugin-md-enhance@2.0.0-beta.17`
